test/e2e/scalability/density.go

```diff
@@ -275,7 +299,10 @@ func cleanupDensityTest(dtc DensityTestConfig) {
 			framework.ExpectNoError(err)
 		}
 	}
+	podCleanupPhase.End()
 
+	auxCleanupPhase := testPhaseDurations.StartPhase(910, "secrets, config maps and daemon sets cleanup")
+	defer auxCleanupPhase.End()
 	// Delete all secrets, configmaps and daemons.
```

Comment:
Maybe move this to separate functions too? Like you did for creation.

Reply:
Done.